wp-content/themes/wp-starter/blocks/navigation-container/render.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * Block: Navigation Container
+ *
+ * @global array $block
+ *
+ * @package WPStarter
+ */
+
+$attrs = [];
+
+?>
+<div <?php block_attrs( $block, 'wp-block-group navigation-container flex flex-col items-end gap-5 md:w-auto w-full', $attrs ); ?>
+	<?php if ( ! is_admin() ) : ?>
+		x-data="{menuIsOpen : false}"
+		x-trap="menuIsOpen"
+	<?php endif; ?>
+>
+	<button
+		<?php if ( ! is_admin() ) : ?>
+			@click="menuIsOpen = !menuIsOpen"
+			x-cloak
+			class="md:hidden absolute inset-y-0 right-0 flex justify-center items-center w-32 h-32 text-gray-900 dark:text-white"
+		<?php else : ?>
+			class="hidden"
+		<?php endif; ?>
+		aria-rel="mobile-toggle"
+	>
+		<svg
+			<?php if ( ! is_admin() ) : ?>
+				x-show="!menuIsOpen"
+			<?php endif; ?>
+			class="m-2" xmlns="http://www.w3.org/2000/svg" width="27" height="18" viewBox="0 0 27 18" fill="currentColor" class="menu-open">
+			<title>Open Menu</title>
```

Comment:
```suggestion
			<title><?php esc_html_e( 'Open Menu', 'wp-starter' ); ?></title>
```
Make this translatable

Reply:
Thanks for catching that!